cristin-commons/src/main/java/no/unit/nva/cristin/common/ErrorMessages.java

```diff
@@ -56,9 +60,19 @@ public class ErrorMessages {
         "No supported fields in payload, not doing anything";
     public static final String ONLY_SUPPORT_BOOLEAN_VALUES = "Valid values are true or false";
     public static final String UPSTREAM_BAD_REQUEST_RESPONSE = "Upstream returned 400 with response body: ";
+    public static final String UPSTREAM_RETURNED_BAD_REQUEST =
+        "Upstream returned 400 (Bad Request). That might indicate bad query parameters";
+
+    /**
+     * Formats and emits a message with valid parameter names.
+     *
+     * @param missing list of valid parameter names
+     * @return formatted string containing a list of valid parameters
```

Comment:
Here you say list of valid parameters, but the code says list of required parameters. There are more parameters that are valid than parameters that are required

Reply:
I though "gyldige verdier", but required is better.